src/cubi_tk/sodar/lz_create.py

```diff
@@ -109,10 +92,10 @@ def execute(self) -> typing.Optional[int]:
             lz = existing_lzs[-1]
         else:
             lz = api.landingzone.create(
-                sodar_url=self.args.sodar_url,
+                sodar_url=self.args.sodar_server_url,
                 sodar_api_token=self.args.sodar_api_token,
                 project_uuid=self.args.project_uuid,
-                assay_uuid=self.args.assay,
+                assay_uuid=self.args.assay_uuid,
```

Comment:
More of a note for future refactoring: here it is assumed that api.landingzone.create can handle 'assay_uuid = None' (and presumably fail on multiple assays). I think this is something we should change in the future (i.e. make sure assay_uuid isn't None before calling the API).